docs/server/source/quickstart.md

```diff
@@ -6,6 +6,20 @@ A. Install MongoDB as the database backend. (There are other options but you can
 
 [Install MongoDB Server 3.4+](https://docs.mongodb.com/manual/tutorial/install-mongodb-on-ubuntu/)
 
+Default database directory where mongodb stores its data: /data/db on Linux and OS X, \data\db on Windows.
+
+If you want to run mongodb with default database path i.e. /data/db, Open a Terminal and run the following commands as root/admin user:
```

Comment:
Some minor formatting things:

1. This is a Quickstart so we're just going to tell them what to do, with no choices. So you can start this line with, "To run..."
1. In prose documentation, we try to write everything with the proper capitalization and branding, so for example we write "MongoDB" rather than "mongodb" (the form one sees in code etc.)
1.  The word "Open" here doesn't begin a new sentence so it can be written lowercase as "open"

Also, you wrote "as root/admin" user but then there is no sudo so I am confused.